build/build-image/Dockerfile

```diff
@@ -28,5 +28,9 @@ RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.
 
 RUN go get -u github.com/golang/lint/golint
 
+RUN git clone https://github.com/duglin/vlinker.git /vlinker
+
+ENV PATH=$PATH:/vlinker/bin
+
```

Comment:
You would think that... but if I had a nickel for every time I though, "surely, _this_ won't add much heft to my image" and then was proven wrong...